wizeline/project-lab
# Sync profiles

Sync Profiles from Wizeline OS

#### What does this PR do?
Added the task to retrieve the Profiles from Wizeline OS and persist them in our DB

#### Where should the reviewer start?

you should start from tasks/syncAllDataFromWOS.ts and tasks/syncAllDataFromWOSTask.ts

#### How should this be manually tested?

you can run tests from syncAllDataFromWOs.test.ts using npm run test
also for the complete task to perform you can run npm run  sync-all-from-wos

After test no data should be present in DB
After the process sync.... all the records from WOS should be in the DB
#### Any background context you want to provide?
Previously we already sync job titles, skills and locations, but now we add profiles in the same task
You should have the WOS keys in your .env or .env.local

WOS_AUTH_API_URL=https://wos-dev.auth0.com/oauth/token
WOS_AUTH_API_AUDIENCE="*****"
WOS_AUTH_API_CLIENT_ID=*********
WOS_AUTH_API_CLIENT_SECRET=*****
WOS_API_URL=****

if you need those values please ask me
#### What are the relevant tickets?

closes #25 

#### Screenshots

 
#### Questions
 
#### Checklist
